templates/zerver/help/scim.md

```diff
@@ -1,62 +1,88 @@
 # SCIM provisioning
 
-SCIM (System for Cross-domain Identity Management) is an standard
+{!admin-only.md!}
+
+SCIM (System for Cross-domain Identity Management) is a standard
 protocol used by Single Sign-On (SSO) services and identity providers
-to provision/deprovision user accounts and groups. Zulip's SCIM
-integration is currently beta and has a few limitations:
+to provision/deprovision user accounts and groups.
+
+Zulip supports SCIM integration, both for Zulip Cloud and self-hosted Zulip
+servers. This page describes how to configure SCIM in Okta.
+
```

Comment:
I think this needs to be "... in Okta for Zulip Cloud." Otherwise, the next sentence doesn't make sense.